Source/Core/DiscIO/VolumeVerifier.h

```diff
@@ -200,6 +200,7 @@ class VolumeVerifier final
 
   u64 m_biggest_referenced_offset = 0;
   u64 m_biggest_verified_offset = 0;
+  bool m_smaller_than_expected = 0;
```

Comment:
Did you mean to initialize a `bool` with zero?

Reply:
Oops, it should be `false`, of course :)